fab/inventory/icds

```diff
@@ -39,6 +39,37 @@
 [control]
 10.247.24.20
 
+[pg2]
+10.247.24.22 devices='["/dev/sdb"]' partitions='["/dev/sdb1"]' datavol_device='/dev/mapper/consolidated-data' hot_standby_server='10.247.24.28'
```

Comment:
- [ ] To update actual devices after partition or don't need it if doing without extra storage